system_test/arrow_fuzz_common.py

```
@@ -0,0 +1,1305 @@
+    sys.float_info.max,
+    -sys.float_info.max,
+    5e-324,
+]
+
+EDGE_STRINGS = [
+    "",
+    "a",
+    "ascii",
+    "日本語",
+    "🚀🌟",
+    "​﻿",
```

Ruff (0.15.14)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)